app/controllers/users/invitations_controller.rb

```diff
@@ -22,6 +25,10 @@ def return_user_to_invite_page
     respond_with_navigational(resource) { render :new }
   end
 
+  def set_target_organisation
+    @target_organisation = Organisation.find_by(id: params[:id] || invite_params[:organisation_id])
```

Comment:
This doesn't need to be `find_by(id: ID)`, if you are doing a lookup by id then `.find(ID)` is better.